src/sentry/receivers/features.py

```diff
@@ -132,6 +132,10 @@ def record_event_processed(project, event, **kwargs):
     FeatureAdoption.objects.bulk_record(project.organization_id, feature_slugs)
 
 
+event_processed.connect(record_event_processed, weak=False)
```

Comment:
Longer term would it make sense to rename this to 'error_processed'?

Reply:
Yeah that might be a more intuitive name